subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -1391,6 +1286,73 @@ By default, Gradle caches changing modules for 24 hours. To change how long Grad
+
+While the strategies introduced above are usually enough to solve most conflicts, Gradle provides more fine-grained mechanisms to resolve version conflicts:
+
+* Configuring a first level dependency as _forced_. This approach is useful if the dependency in conflict is already a first level dependency. See examples in api:org.gradle.api.artifacts.dsl.DependencyHandler[].
+* Configuring any dependency (transitive or not) as _forced_. This approach is useful if the dependency in conflict is a transitive dependency. It also can be used to force versions of first level dependencies. See examples in api:org.gradle.api.artifacts.ResolutionStrategy[]
+* Configuring dependency resolution to _prefer modules that are part of your build_ (transitive or not). This approach is useful if your build contains custom forks of modules (as part of <<multi_project_builds>> or as include in <<composite_builds>>). See examples in api:org.gradle.api.artifacts.ResolutionStrategy[].
+* Dependency resolve rules are an <<feature_lifecycle,incubating>> feature introduced in Gradle 1.4 which give you fine-grained control over the version selected for a particular dependency.
+
+To deal with problems due to version conflicts, reports with dependency graphs are also very helpful. Such reports are another feature of dependency management.
+
+[[sub:dynamic_versions_and_changing_modules]]
+==== Use Dynamic Versions and Changing Modules
+
+There are many situations when you want to use the latest version of a particular dependency, or the latest in a range of versions. This can be a requirement during development, or you may be developing a library that is designed to work with a range of dependency versions. You can easily depend on these constantly changing dependencies by using a _dynamic version_. A dynamic version can be either a version range (e.g. `2.+`) or it can be a placeholder for the latest version available (e.g. `latest.integration`).
+
+Alternatively, sometimes the module you request can change over time, even for the same version. An example of this type of _changing module_ is a Maven `SNAPSHOT` module, which always points at the latest artifact published. In other words, a standard Maven snapshot is a module that never stands still so to speak, it is a “changing module”.
+
+The main difference between a _dynamic version_ and a _changing module_ is that when you resolve a _dynamic version_, you'll get the real, static version as the module name. When you resolve a _changing module_, the artifacts are named using the version you requested, but the underlying artifacts may change over time.
+
+By default, Gradle caches dynamic versions and changing modules for 24 hours. You can override the default cache modes using <<sec:cache_command_line_options,command line options>>. You can change the cache expiry times in your build using the resolution strategy (see <<sec:controlling_caching>>).
```

Comment:
There must be something in the build environment or init scripts for this. Better to link to that for configuring this than command line options.

Reply:
"You can change the cache expiry times in your build using the resolution strategy (see <<sec:controlling_caching>>)."

to 

"You can also change cache expiry times in your build <<sec:controlling_caching, using a ResolutionStrategy>>."

Reply:
This is unchanged content. It was planning rework the content in a different PR.